pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -1569,6 +1569,17 @@ func Convert_v1_VirtualMachine_To_api_Domain(vmi *v1.VirtualMachineInstance, dom
 		}
 	}
 
+	if virtLauncherLogVerbosity, err := strconv.Atoi(os.Getenv(services.ENV_VAR_VIRT_LAUNCHER_LOG_VERBOSITY)); err == nil && (virtLauncherLogVerbosity > services.EXT_LOG_VERBOSITY_THRESHOLD) {
+
+		initializeQEMUCmdAndQEMUArg(domain)
+
+		domain.Spec.QEMUCmd.QEMUArg = append(domain.Spec.QEMUCmd.QEMUArg,
+			api.Arg{Value: "-chardev"},
+			api.Arg{Value: "file,id=firmwarelog,path=/tmp/qemu-firmware.log"},
```

Comment:
@iholder-redhat I just hope we could follow up in another PR and move this log to be seen at part of the virt-launcher log. 
Something like we do for the QEMU log[1], but we could actually do it better... see[2]

[1] https://github.com/kubevirt/kubevirt/blob/master/pkg/virt-launcher/virtwrap/util/libvirt_helper.go#L293
[2] https://www.seabios.org/Debugging (Timing debug messages
)

Reply:
Sounds good :)